packages/playground/windows/playground/Playground.vcxproj

```diff
@@ -43,10 +43,7 @@
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
-  <PropertyGroup>
-    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
-  </PropertyGroup>
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactPackageDirectories.props" />
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactDirectories.props" />
```

Comment:
I'm assuming this is the same as `Playground-win32.vcxproj`.

Reply:
Yep. Once these are converted from ReactUwp to Microsoft.ReactNative we can keep everything tucked into vnext as well.